### ISSUE TITLE
Receive error in server console after placing an order for a single shop in multi-shop mode

### ISSUE THREAD

**Issue by zenweasel** [reactioncommerce/reaction#3024]
### Expected behavior
No errors

### Actual Behavior
Receive this error in console:
```
Exception while invoking method 'orders/refunds/list' TypeError: Cannot read property 'paymentMethod' of undefined
    at [object Object].ordersRefundsList (server/methods/core/orders.js:953:27)
    at packages/check.js:128:16
    at [object Object].EVp.withValue (packages/meteor.js:1134:15)
    at Object.exports.Match._failIfArgumentsAreNotAllChecked (packages/check.js:127:41)
    at maybeAuditArgumentChecks (packages/ddp-server/livedata_server.js:1765:18)
    at packages/ddp-server/livedata_server.js:719:19
    at [object Object].EVp.withValue (packages/meteor.js:1134:15)
    at packages/ddp-server/livedata_server.js:717:46
    at [object Object].EVp.withValue (packages/meteor.js:1134:15)
    at packages/ddp-server/livedata_server.js:715:46
    at [object Object]._.extend.protocol_handlers.method (packages/ddp-server/livedata_server.js:689:23)
    at packages/ddp-server/livedata_server.js:559:43
```

This seems to be because this function is returning the paymentMethod for the wrong shop (only the current shop)
https://github.com/reactioncommerce/reaction/blob/0059da1f76b8c51930eedc94a4fd4e9b0c1806ce/server/methods/core/orders.js#L18

### Steps to Reproduce the Behavior
1. Create a marketplace shop and create a product for that shop
1. Place only that product in the cart and checkout
1. Observe the error

### Versions
```
Node: 7.10.0
NPM: 5.3.0
Meteor Node: 4.8.4
Meteor NPM: 4.6.1
Reaction CLI: 0.13.1
Reaction: 1.5.0
Reaction branch: brent-fix-issue-2836
Docker: 17.09.0-ce
```

**Comment by spencern** [comment]
What's the status on this?

**Comment by kieha** [comment]
Started looking into this but put it on hold to wrap up with #2832. I'll remove my assignment just so someone else not currently working on anything could pick it up, but I'll get back to it once done with my current task if it'll still be unassigned.

**Comment by spencern** [comment]
@kieha how's this issue coming along?

**Comment by kieha** [comment]
Update: 

-If you're in shop A, and you buy a product from shop B, and then complete the order in shop A, this function https://github.com/reactioncommerce/reaction/blob/0059da1f76b8c51930eedc94a4fd4e9b0c1806ce/server/methods/core/orders.js#L18
returns undefined as the shopId of that order's billing record is that of shop B, hence the error.

Solution:
- returning an empty object if the function returns `undefined`, and build up variables that won't throw further errors.

**Comment by spencern** [comment]
@kieha will you post an update here about what was decided with this issue? Based on this morning's conversation, it seems like we're ready to resolve it either now or with minimal changes.

**Comment by kieha** [comment]
@spencern this is being partially resolved with minimal changes, i.e. iterating over the billing and shipping records and getting the appropriate information from that to use in the email notification. However, a separate issue was opened here #3065 to make the notifications multi-shop aware.

**Comment by spencern** [comment]
Resolved by #3064
